vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -35,8 +37,25 @@ void AccessibilityInfo::isTouchExplorationEnabled(
   onSuccess(UiaClientsAreListening());
 }
 
-void AccessibilityInfo::setAccessibilityFocus(double /*reactTag*/) noexcept {
-  // no-op - This appears to be unused in RN
+void AccessibilityInfo::setAccessibilityFocus(double reactTag) noexcept {
```

Comment:
reactTag should be an int, any idea why this is marked as a double?

Reply:
I know that the JS setAccessibilityFocus API is expecting a parameter of type "number". In skimming over the Android code it seems that they treat all reactTags in the native code as int so I'm not sure why it's marked as double.